keyboards/lily58/keymaps/yshrsmz/keymap.c

```diff
@@ -0,0 +1,237 @@
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```

Comment:
It might be a good idea to convert these to an enum, so the layers get numbered automatically. eg:
```c
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST
};
```